src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlLogger.cs

```diff
@@ -273,11 +274,21 @@ public void TestRunCompleteHandler(object sender, TestRunCompleteEventArgs e)
                 PassPercentage = (PassedTests * 100) / TotalTests,
                 TotalRunTime = GetFormattedDurationString(e.ElapsedTimeInRunningTests),
             };
-            var isLogFileNameParameterExists = parametersDictionary.TryGetValue(HtmlLoggerConstants.LogFileNameKey,
-                               out string logFileNameValue);
-            if (isLogFileNameParameterExists && !string.IsNullOrWhiteSpace(logFileNameValue))
+
+            if (this.parametersDictionary.TryGetValue(HtmlLoggerConstants.LogFilePrefixKey, out string logFilePrefixValue) && !string.IsNullOrWhiteSpace(logFilePrefixValue))
+            {
+
+                var framework = this.parametersDictionary[DefaultLoggerParameterNames.TargetFramework] ?? string.Empty;
+                framework = NuGetFramework.Parse(framework).GetShortFolderName();
```

Comment:
What happens when framework is null/empty

Reply:
Throws exception if framework is empty. so proceeding with this step only when framework is not null